src/ast.js

```diff
@@ -3,24 +3,50 @@ const {symbolMap} = require('./constants');
 
 function traversSourceCode(source, parse) {
     const root = AST.parse(source, parse);
+    const sourceLines = source.split('\n');
     const nodes = [];
     let stack = [root];
 
+    function getSrc({start, end}) {
```

Comment:
Rename `start` to `startLine`
Rename `end` to `endLine`

Reply:
Start and end are objects with line and column attributes, so I think adding Line to the name will create confusion